grafana.yaml

```diff
@@ -0,0 +1,41 @@
+version: "3"
+services:
+  grafana:
+    image: grafana/grafana-oss:8.5.1
```

Comment:
If we wanna be really specific, we might want to include the registry, i.e. docker.io/library/grafana/grafana-oss . This would set a bit of a signal that you/we could self-host images in the future or use quay.io or other registries.